auditlog/admin-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+# Configuration file
+# key = value
+
+quarkus.http.port=8085
```

Comment:
For the sake of consistency with the other two services: can you remove this here, and instead change the exposed port of the service in the Compose file and override it in the README in the "Running locally" section?

Reply:
removed and readme update